platform/src/main/java/org/stellar/anchor/platform/rpc/NotifyCustomerInfoUpdatedHandler.java

```diff
@@ -59,15 +60,29 @@ public RpcMethod getRpcMethod() {
   @Override
   protected SepTransactionStatus getNextStatus(
       JdbcSepTransaction txn, NotifyCustomerInfoUpdatedRequest request) {
+    if (SEP_6 == Sep.from(txn.getProtocol())) {
+      switch (Sep12Status.valueOf(request.getStatus())) {
+        case ACCEPTED, PROCESSING:
+          return PENDING_ANCHOR;
+        case NEEDS_INFO:
+          return PENDING_CUSTOMER_INFO_UPDATE;
+        case REJECTED:
+          return ERROR;
+      }
+    }
     return PENDING_RECEIVER;
   }
 
   @Override
   protected Set<SepTransactionStatus> getSupportedStatuses(JdbcSepTransaction txn) {
-    if (SEP_31 == Sep.from(txn.getProtocol())) {
-      return Set.of(PENDING_CUSTOMER_INFO_UPDATE);
+    switch (Sep.from(txn.getProtocol())) {
```

Comment:
In which status we start before we call the notify_customer_info_updated first time? Is it PENDING_CUSTOMER_INFO_UPDATE?

Reply:
It should be according to the SEP-31 status transition diagram.

Reply:
Oh sorry asking for SEP-6

Reply:
For SEP-6, it's also PENDING_CUSTOMER_INFO_UPDATE.